product_variant_change_attribute_value/wizards/product_variant_attribute_value_action.py

```diff
@@ -8,9 +8,16 @@ class ProductVariantAttributeValueAction(models.TransientModel):
     _name = "variant.attribute.value.action"
     _description = "Wizard action to do on variant attribute value"
 
+    def _get_attibute_action_list(self):
+        return [
+            ("delete", "Delete"),
+            ("replace", "Replace"),
+            ("do_nothing", "Do Nothing"),
+        ]
+
     product_attribute_value_id = fields.Many2one("product.attribute.value",)
     attribute_action = fields.Selection(
-        [("delete", "Delete"), ("replace", "Replace"), ("do_nothing", "Do Nothing")],
+        selection=lambda self: self._get_attibute_action_list(),
```

Comment:
`selection="_selection_attribute_action"`

Reply:
I changed it